50SAT/SPEX$50SAT.tex

```diff
@@ -0,0 +1,346 @@
+  \item Test procedures and results for key areas including radio performance, thermal and structural tolerance, and software qualifications
+\end{itemize}
+The published designs also include an arduino-based radio receiver station. The team could develop this component as well, or preferably, leverage other teams and their projects to satisfy this requirement.
+Finally, the team will need to generate a presentation poster that could be used at ImagineRIT or other public SPEX events to highlight the project.
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+If this project is conducted over a semester timeline, the team will need to expediently cover the required areas defined below.
+
+This project will take place over three technical domains: electrical, mechanical, and software.
+
+For the electrical systems, the team will need to review the existing PCB designs, primarily to determine whether it is feasible to manufacture and populate them on campus.
+Also the team will need to determine if all components required are still available, and if not, find suitable replacements and make the necessary design changes.
+PCB assembly will be the system with the longest lead time and should be prioritized first to ensure no bottlenecks on the critical path.
+Once assembled, each will need to be tested, and integrated within the system structure.
+
+For the mechanical systems, the team should evaluate the feasibility of manufacturing the frame on-campus.
+Any tweaks to the internal structure and placement of internal components should be locked down as early as possible.
+The mechanical team will also be responsible for testing the frame and whole assembly for verification against thermal and vibration requirements.
+
+For the onboard software, the team should review the existing code, and try to demonstrate functionality on similar hardware to prevent dependency on the electrical team.
+Once the primary electronics are ready, the software should be tested for functionality on the system hardware.
+
+The timeline below is preliminary:
+\begin{table*}
+  \caption{Estimated Timeline}
+  \small\centering
+  \begin{tabular}{@{}lll@{}} \toprule
+    Phase & Task & Duration \\ \midrule
+    1 & Review the existing \$50SAT designs and materials &  no more than 2 weeks \\
+    2 & Subsystem development & 6 weeks \\
+    2 & Order PCB design and/or assembly & 6 weeks \\
+    2 & Review changes to mechanical strucuture and order materials & no more than 2 weeks \\
+    2 & Testing of individual subsystems & 2 weeks \\
+    3 & System Assembly & 1 week  \\
+    4 & System testing & 2 weeks  \\
+    5 & Generate documentation and delivery to SPEX & 1 week  \\
+    \bottomrule
+  \end{tabular}
+\label{tab:timeline}
+\end{table*}
+
+\begin{itemize}
```

Comment:
1. I hadn't pulled changes locally. Table looks much better!! We might be able to do something clever to get rid of the "Phase" column. Without the phase column, I think it would fit in a single col

2.  It's a PDF interrnal link, intended. Same thing happens when you make citations. It shouldn't show up in print or in anything other than Adobe Acrobat